src/client/src/rt-components/tear-off/DesktopWindow.ts

```diff
@@ -1,7 +1,11 @@
+import logdown from 'logdown'
 import { WindowConfig } from './types'
 
 type DesktopWindowProps = WindowConfig
 
+const LOG_NAME = 'DesktopWindow:'
```

Comment:
Similarly this is not a useful convention here as we won't be referencing a module global in any of our methods.